docs/advanced-reading/fetchers.md

```diff
@@ -14,6 +14,42 @@ Fetchers are the implementation of the [search using online services](https://do
 
 On Windows, you have to log-off and log-on to let IntelliJ know about the environment variable change. Execute the gradle task "processResources" in the group "others" within IntelliJ to ensure the values have been correctly written. Now, the fetcher tests should run without issues.
 
+## Fetcher prioritization
+
+- all fetchers run in parallel
+- the result with the highest priority wins
+- `InterruptedException` | `ExecutionException` | `CancellationException` are ignored
+
+### Trust Levels
+
+- SOURCE (highest)
+- PUBLISHER
```

Comment:
It would be good if these trust levels are described in more detail, i.e. what are the criteria.